### ISSUE TITLE
Assign Field to specific column

### ISSUE THREAD

**Issue by Samoht1** [django-import-export/django-import-export#1358]
I searched far and wide but sadly couldn't find any solution. I have a working form which exports all data to xls using Resource and Fields. But now I need to go a bit further and make it a bit more advanced. I need to export all data to an existing Excel template, assigning each field to a specific column, as in, for instance: Field 1 needs to be filled in B3, Field 2 in C4 etc. etc. Is such a thing possible with this package? Or do I need to start over and use xlswriter or something?

**Comment by matthewhegarty** [comment]
`django-import-export` is exporting a ResultSet as row data (e.g. csv, XLS etc), so by default it's not going to be possible to put field 1 in to B3 etc.  However, there are export hooks such as [`after_export()`](https://django-import-export.readthedocs.io/en/latest/api_resources.html?highlight=after_export#import_export.resources.Resource.after_export) where you can manipulate the data to be returned.  

You don't say whether you are using the Admin site integration, but if so, there are other hooks you can override to customise the output.

In short, `django-import-export` is going to give you your data back as rows, so you can further customise output as desired.

**Comment by Samoht1** [comment]
sadly the docs don't explain that method at all so I have no idea how to use it...

**Comment by matthewhegarty** [comment]
If you have a specific question, feel free to post to [SO](https://stackoverflow.com/questions/tagged/django-import-export) and I'll help if I can.

**Comment by Samoht1** [comment]
https://stackoverflow.com/questions/70360130/django-import-export-assign-field-to-specific-column/70380872#70380872
